dspy/predict/predict.py

```diff
@@ -1,21 +1,20 @@
+from functools import lru_cache
 import logging
 import random
-from functools import lru_cache
 
 from pydantic import BaseModel
 
 import dsp
 from dspy.predict.parameter import Parameter
-from dspy.primitives.prediction import Prediction
 from dspy.primitives.program import Module
+from dspy.primitives.prediction import Prediction
```

Comment:
Is this order change expected? the old one seems to sort correctly

Reply:
Thank you for the catch, reverted